09Ianuarie/src/ro/sci/bv/CalculateMax/CalculateMax.java

```diff
@@ -0,0 +1,39 @@
+/**
+ * This program asks the user to type 3 numbers, in order to find the biggest of them
+ * Then the program will print it to the user
+ *
+ * @author Bogdan Butuza
+ */
```

Comment:
the javadoc should be before the class declaration not before the package or imports.